### PR TITLE
common: send temp for abs and diff press

### PR DESCRIPTION
**Pull request by IamPete1**
A possible method for returning the airspeed temperature along with the baro temp. @WickedShell 

https://github.com/ArduPilot/ardupilot/pull/14852

### CODE REVIEW

File: message_definitions/v1.0/common.xml [hunk 1]
```diff
@@ -3668,7 +3668,9 @@
       <field type="uint32_t" name="time_boot_ms" units="ms">Timestamp (time since system boot).</field>
       <field type="float" name="press_abs" units="hPa">Absolute pressure</field>
       <field type="float" name="press_diff" units="hPa">Differential pressure 1</field>
-      <field type="int16_t" name="temperature" units="cdegC">Temperature</field>
+      <field type="int16_t" name="temperature_press_abs" units="cdegC">Temperature</field>
```
Comment (WickedShell) [inline message_definitions/v1.0/common.xml:3671]: you can't rename an exsisting field (or change the type actually), it breaks all compatability

Comment (IamPete1) [reply message_definitions/v1.0/common.xml]: fixed

**Comment by auturgy** [comment]
Changes to common should go via upstream.  fyi the high_latency messages include airspeed temp.

**Comment by tridge** [comment]
looks good to me. I'll merge
@IamPete1 can you do a PR for upstream?

**Comment by auturgy** [comment]
Can we please refrain from introducing divergence in common.xml?

**Comment by amilcarlucas** [comment]
It would be great if such changes would get merged upstream first

**Pull request merged**